package.json

```diff
@@ -17,7 +17,8 @@
     "test:routes": "NODE_ENV=mocha LOG_LEVEL=OFF mocha --compilers ts:ts-node/register -r tsconfig-paths/register $(find src/test \\( -name '*.ts' -or -name '*.js' \\) -path '*/routes/*') --reporter mochawesome --reporter-options reportFilename=routes,inlineAssets=true,reportTitle=legal-rep-frontend-routes",
     "test:a11y": "mocha --compilers ts:ts-node/register -r tsconfig-paths/register src/test/a11y --timeout 60000 --reporter mochawesome --reporter-options reportFilename=a11y,inlineAssets=true,reportTitle=legal-rep-frontend-a11y",
     "test:nsp": "nsp check",
-    "test:coverage": "cross-env NODE_PATH=. nyc istanbul cover  ./node_modules/mocha/bin/_mocha -- --compilers ts:ts-node/register -r tsconfig-paths/register $(find src/test \\( -name '*.ts' -or -name '*.js' \\) ! -path '*a11y*' ! -path '*/routes/*')",
+    "test:coverage": "cross-env NODE_PATH=. NODE_ENV=mocha nyc istanbul cover mocha -- --compilers ts:ts-node/register -r tsconfig-paths/register $(find src/test \\( -name '*.ts' -or -name '*.js' \\) ! -path '*a11y*')",
```

Comment:
```
yarn check
```
fails when run on nyc dependency.

It is used to make sure the lock file and package.json file is in sync

T2 is missing the check that enforces it in their CI,
example from T1
https://github.com/hmcts/cmc-citizen-frontend/blob/master/.travis.yml#L6

Reply:
@vianvi This is still failing and if I try and my console gets hammered with warnings like this:
```
warning The case-insensitive file /Users/timja/projects/moj-reform/cmc/cmc-legal-rep-frontend/node_modules/nyc/node_modules/yargs/node_modules/yargs-parser/package.json shouldn't be copied twice in one bulk copy
```